pages/blog/2025-05-21-customizable-dashboards.mdx

```diff
@@ -0,0 +1,138 @@
+---
+title: "How we Built Scalable & Customizable Dashboards"
+date: 2025/05/21
+description: "A technical deep dive into how we built Langfuse's flexible, real-time dashboard system for LLM monitoring."
+tag: engineering
+ogImage: /images/blog/2025-05-19-launch-week-3/lw3-teaser-blog.png
+author: Steffen
+---
+
+import { BlogHeader } from "@/components/blog/BlogHeader";
+
+<BlogHeader
+  title="How we Built Scalable & Customizable Dashboards"
+  description="A technical deep dive into how we built Langfuse's flexible, real-time dashboard system for LLM monitoring"
+  authors={["steffenschmitz"]}
+  image="/images/blog/2025-05-19-launch-week-3/lw3-teaser-blog.png"
+  date="May 21, 2025"
+/>
+
+On Day 3 of [Langfuse Launch Week 3](/blog/2025-05-19-launch-week-3), we introduce **customizable dashboards** ([docs](/docs/analytics/custom-dashboards.mdx)): a powerful way to visualize LLM usage directly within the Langfuse UI. Whether you want to track latency trends, monitor user feedback, or correlate cost with performance, our new dashboards let you build the charts you need - right where you need them. And for those who prefer working in their own analytics stack, we've made the same querying capabilities available via our API ([docs](/pages/docs/analytics/metrics-api.mdx)).
+
+In this post, I'll take you behind the scenes: from our product thinking to the technical implementation, testing, and rollout—and share the lessons we learned in building flexible, real-time insights into your LLM pipelines.
+
+> TODO: add launch week demo video
+
+## Motivation
```

Comment:
currently the chapters seem to be a bit overlapping, probably makes sense to group the correctly (motivation, architecture, delivery)